clinica/iotools/bids_utils.py

```diff
@@ -467,15 +467,20 @@ def _write_bids_dataset_description(
         bids_desc.write(to=f)
 
 
-def _write_readme(bids_dir: Union[str, Path]) -> None:
-    """Write `README`file at the root of the BIDS directory."""
-    import clinica
+def _write_readme(
+    study_name: str,
+    data_dict: dict,
+    bids_dir: Union[str, Path],
+    bids_version: Optional[str] = None,
+) -> None:
+    """Write `README` at the root of the BIDS directory."""
+    from clinica.iotools.bids_readme import BIDSReadme
 
+    bids_desc = BIDSReadme(
```

Comment:
Might as well use the same name. It's a readme, let the variable named readme too.

```suggestion
    bids_readme = BIDSReadme(
```